.npmignore

```diff
@@ -0,0 +1,4 @@
+/.*/
+/.*
```

Comment:
If I get it right, the first pattern specify all **root directories** that starts with a dot `.`, and the second pattern specifies all **root files** that starts with a dot `.`?

Shouldn't we also leave out `karma.conf.js`?